### DISCUSSION TITLE
Set account as personal or business

### DISCUSSION THREAD

**Discussion by vanillajonathan** [NickvisionApps/Denaro#120] [Ideas]
Maybe there could be a way to set an account as personal for your own finance, or as business for your company.

**Pros**: It lets users differentiate accounts used for different purposes and in different contexts.
**Cons**: Maybe it can lead to confusion among users regarding whether it has any impact or consequences.

**Comment by fsobolev** [comment]
@zothma another use-case for metainfo? :) We'll need to add some kind of account creation dialog anyway if there will be possibility to add name to an account, so we could also implement personal/business switch and add label that it's only usability feature that doesn't impact how the app works.

**Comment by zothma** [comment]
I'll work on a `metainfo` table, and maybe some mockups that could add those features. The more I think about it the more it seems a good idea!

**Reply by vanillajonathan** [reply]
Maybe #118 could be in the metainfo too?

**Reply by fsobolev** [reply]
> Maybe #118 could be in the metainfo too?

Well you will be able to add any emoji wherever you want in account name, I'm not sure that a feature to explicitly add flag to account is needed. But what definitely is needed in metainfo IMO is currency (when https://github.com/nlogozzo/NickvisionMoney/issues/93 will be implemented).

**Reply by zothma** [reply]
> Maybe #118 could be in the metainfo too?

I do agree with @fsobolev on the fact that it may just be in the account name if implemented. The aim is to stay simple enough and not bloat with features to comply to everyone, but rather stick to a main goal, as stated in the Gnome HIG

**Reply by vanillajonathan** [reply]
This could be cleanly designed with a `primary-icon` on the `Gtk.Entry`. Then it would look simple.

**Comment by nlogozzo** [comment]
Will be incorporated into #128
